src/assets/css/blogposts.scss

```diff
@@ -64,37 +68,74 @@
     }
   }
 
-  @include respond-to(xl) {
-    max-width: 750px;
+  .wp-block-pullquote {
+    @include font-bold();
+
+    border-left: 3px solid $pull-quote-color;
+    color: $pull-quote-color;
+    font-size: $heading-xs-font-size;
+    line-height: $heading-xs-line-height;
```

Comment:
I missed this before, but these are px based line-heights. It could be good to consider using unitless line-heights see https://meyerweb.com/eric/thoughts/2006/02/08/unitless-line-heights/